docs/installing-northstar/northstar-installers/VTOL-guide.md

```diff
@@ -0,0 +1,101 @@
+# VTOL Information
```

Comment:
filename should be all lowercase "cause GitBook" (it has some weird requirements, idk) ^^